src/content.js

```diff
@@ -131,29 +132,35 @@ async function buildMetaIndex(contentDir, invaderDefsDir, baseUrl, packageVersio
     .join("\n");
 
   const findTagPageByName = (tagName) => {
-    const page = pages.find(page => page._slug == tagName && page.template == "tag");
+    const page = pages.find(page => page.tagName == tagName);
     if (!page) throw new Error(`Failed to find tag page for ${tagName}`);
     return page;
   };
 
   const findToolPageByName = (toolName) => {
-    const page = pages.find(page => page._slug == toolName.toLowerCase() && page.template == "tool");
+    const page = pages.find(page => page.toolName == toolName);
```

Comment:
Should we enforce case-correctness of toolnames? (I'm gravitating towards yes)

Reply:
I actually forgot to delete this function from earlier development. It's unused. When the metabox needs to link to an item involved in a workflow (e.g. `edit`, `from`, `to`, or `using`) or a similar item, it uses `resolvePage` with the page slug found in `tools.yml`.